cypress/e2e/draw/draw-feat-style.cy.ts

```diff
@@ -0,0 +1,105 @@
+describe('Test style edition of Point feature', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    describe('Draw "Point"', () => {
+      cy.get('button[data-cy="drawButton"]').click()
+      cy.get('button[data-cy="drawPointButton"]').click()
+      cy.get('div.ol-viewport').click(400, 300, { force: true })
+    })
+    describe('Edit feature style', () => {
+      cy.get('*[data-cy="featItemActionStyle"]').click()
+      cy.get('*[data-cy="featStyleColor"]')
+        .invoke('val', '#00ff00')
+        .trigger('input')
+      cy.get('*[data-cy="featStyleSymbol"]').click()
+      cy.get('*[data-cy="featStyleSymbol_2"]').click()
+      cy.get('*[data-cy="featStyleSize"]').type('{selectAll}50')
+      cy.get('*[data-cy="featStyleSize"]').trigger('input')
+      cy.get('*[data-cy="featStyleAngle"]').type('{selectAll}30')
+      cy.get('*[data-cy="featStyleAngle"]').trigger('input')
+      cy.get('button[data-cy="featureEditValidate"]').click()
+    })
+  })
+
+  describe('When checking URL', () => {
+    it('feature style shall be encoded', () => {
+      cy.url().as('url')
+      cy.window()
+        .its('featureHash')
```

Comment:
I did not manage to import the functions in cypress, since the package luxembourg does not exist and must be built first. The window trick seems to be common in tips I found online and we already used it for olMap here: https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/blob/main/cypress/support/commands.ts#L51

I found the e2e test via the URL to be very powerful, the encoding in the URL is a good representation of the app's state